src/main/java/org/bahmni/sms/impl/DefaultSmsSender.java

```diff
@@ -52,6 +51,9 @@ public ResponseEntity send(String phoneNumber, String messageText) {
             String jsonObject = Obj.writeValueAsString(smsRequest);
             StringEntity params = new StringEntity(jsonObject);
             request.addHeader("content-type", "application/json");
+            if(smsProperties.getToken().isEmpty()){
+                logger.warn("token doesn't exist in env");
```

Comment:
Along with this logging statement, do add logs after executing the D7 SMS API call.
Without logging the response, we will not be able to know whether the sms has flown out from D7 or not.